docs/website/content/v0.3/en/guides/cloud/aws.md

```diff
@@ -2,18 +2,225 @@
 title: 'AWS'
 ---
 
-First, create the AMI:
+## Creating a Cluster via the AWS CLI
+
+In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
+We assume an existing VPC.
+
+### Create the Subnet
+
+```bash
+aws ec2 create-subnet \
+    --region ${REGION} \
```

Comment:
should we explain what should be the values for all the vars/provide some samples?

Reply:
Examples would be good 👍.